config/config.eg.yaml

```diff
@@ -1,8 +1,14 @@
 database:
   driver: sqlite
   name: data.db
-  loglevel: INFO # SILENT, ERROR, WARN, INFO
+  log:
+    enable: true
```

Comment:
Instead of `enable` it needs to `store`, that `store` gives us a meaningful message. We always `enable` logs but whether it's on console only or console and file both. And it needs to be false by default.

Also, same on the other two loggers as well.